packages/replay/src/replay.ts

```diff
@@ -981,6 +982,9 @@ export class ReplayContainer implements ReplayContainerInterface {
 
     const earliestEvent = eventBuffer.getEarliestTimestamp();
     if (earliestEvent && earliestEvent < this._context.initialTimestamp) {
+      // eslint-disable-next-line no-console
+      const log = this.getOptions()._experiments.traceInternals ? console.info : logger.info;
+      __DEBUG_BUILD__ && log(`[Replay] Updating initial timestamp to ${earliestEvent}`);
```

Comment:
@billyvg added this log here, that's the only place that updates this (apart from initial setup, which sets it to `Date.now()` by default).